src/overlays/actors/ovl_En_Eiyer/z_en_eiyer.h

```diff
@@ -6,9 +6,18 @@
 
 struct EnEiyer;
 
+typedef void (*EnEiyerActionFunc)(struct EnEiyer *, GlobalContext*);
+
 typedef struct EnEiyer {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x188];
+    /* 0x014C */ SkelAnime skelanime;
+    /* 0x0190 */ EnEiyerActionFunc actionFunc;
+    /* 0x0194 */ s16 actionTimer;
```

Comment:
Since there's only one timer in here, could just call it `timer`. Might also be worth documenting what setting it to `-1` does, since it's pretty unusual.